packages/salesforcedx-vscode-lightning/test/telemetry/telemetry.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 import { assert, match, SinonStub, stub } from 'sinon';
 import TelemetryReporter from 'vscode-extension-telemetry';
-import { TelemetryService } from '../../../src/telemetry/telemetry';
+import { TelemetryService } from '../../src/telemetry/telemetry';
```

Comment:
I thought the telemetry tests didn't require vscode to run since we mocked some of the needed vscode apis.